frontend/app/components/wp-table/wp-table.directive.html

```diff
@@ -131,7 +131,8 @@
             </accessible-checkbox>
             <span>
               <a class="wp-table--details-link"
-                 ui-sref="work-packages.list.details.overview({workPackageId: row.object.id})">
+                 ui-state="desiredSplitViewState"
+                 ui-state-params="{workPackageId: row.object.id}">
```

Comment:
Yes, and a watcher. So if we need the "true" href, this is a good solution. If we don't need the href, ng-click would reduce the watcher count.